src/lib.rs

```diff
@@ -154,7 +152,8 @@ pub fn run<I: BufRead, O: Write + Send>(input: I, output: O, config: Config) ->
 
     let theme_opt = config
         .theme
-        .or_else(|| header.term_theme.map(Theme::Embedded))
+        .clone()
+        .or_else(|| header.term_theme.clone().map(Theme::Embedded))
```

Comment:
Clones are needed because this is now accessing config and header through shared refs, previously it moved out of them.